--- configure.orig	2006-11-02 23:55:28.000000000 +0200
+++ configure	2006-11-02 23:56:09.000000000 +0200
@@ -1819,9 +1819,9 @@
 fi
 
 
-PACKAGE=1.6 dist-bzip2
+PACKAGE=$PACKAGE_NAME
 
-VERSION=
+VERSION=$PACKAGE_VERSION
 
 if test "`cd $srcdir && pwd`" != "`pwd`" && test -f $srcdir/config.status; then
   { { echo "$as_me:$LINENO: error: source directory already configured; run \"make distclean\" there first" >&5
